influxql/iterator.gen.go.tmpl

```diff
@@ -466,7 +466,7 @@ func (itr *{{$k.name}}ParallelIterator) Close() error {
 func (itr *{{$k.name}}ParallelIterator) Next() (*{{$k.Name}}Point, error) {
 	v, ok := <-itr.ch
 	if !ok {
-		return nil, io.EOF
+		return nil, nil
```

Comment:
Possibly, but I don't think a single one of our iterators returns `io.EOF`. Most just return `nil` when they are finished.